### PR TITLE
CLDSRV-148 - test alerts

### PR DESCRIPTION
**Pull request by lavigne958**
Run action `action-prom-render-test` to test alerts

### CODE REVIEW

File: .github/workflows/alerts.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Render and test ${{ matrix.tests.name }}
+        uses: scality/action-prom-render-test@1.0.1
+        with:
+          alert_file_path: monitoring/alerts.yaml
+          test_file_path: ${{ matrix.tests.file }}
+          alert_inputs: >-
+            namespace=zenko,service=artesca-data-connector-s3api-metrics,replicas=3,systemErrorsWarningThreshold=0.03,systemErrorsCriticalThreshold=0.05,listingLatencyWarningThreshold=0.300,listingLatencyCriticalThreshold=0.500,deleteLatencyWarningThreshold=0.500,deleteLatencyCriticalThreshold=1.000
```
Comment (francoisferrand) [inline .github/workflows/alerts.yaml]: should not need to specify all inputs : just `namespace`, `service` and `replicas` should be OK.

Comment (francoisferrand) [reply .github/workflows/alerts.yaml]:
```suggestion
            namespace=zenko,service=artesca-data-connector-s3api-metrics,replicas=3
```

Comment (lavigne958) [reply .github/workflows/alerts.yaml]: done.

**Review by ghivert — APPROVED** [review]

**Review by francoisferrand — APPROVED** [review]

**Pull request merged**